tests/python/unittest/test_gluon_rnn.py

```diff
@@ -67,6 +67,22 @@ def test_lstm_forget_bias():
                                forget_bias * np.ones(100, ), np.zeros((2 * 100,))])
     assert_allclose(mod.get_params()[0][bias_argument].asnumpy(), expected_bias)
 
+def test_lstm_cpu_inference():
+    # should behave the same as lstm cell
+    atol = 1e-6
+    x = mx.nd.ones(shape=(2, 2, 2))
+    model = mx.gluon.nn.Sequential()
+    with model.name_scope():
+        model.add(mx.gluon.rnn.LSTM(2, num_layers=6, bidirectional=True))
+    model.initialize(mx.init.One())
```

Comment:
Could you also test the consistency between cpu and gpu, with same random weights and random inputs?

Reply:
will it break CPU tests?

Reply:
You can use this function to test consistency.
https://github.com/apache/incubator-mxnet/blob/master/python/mxnet/test_utils.py#L1207

Reply:
Under the current input and weight, your test would still pass even if the weights are iterated backwards. Unfortunately it's not in an acceptable state.